index.html

```diff
@@ -1,14 +1,55 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://kit.fontawesome.com/9c6e5a05ae.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Anaheim&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
     <title>Document</title>
 </head>
+
 <body>
+    <section class="section-todo">
+        
+            <div class="rest-todo">
+
+
+                <img class="moon-img"  src="./img/Vector.png" alt="">
+                <span id="text-rest">Reset</span>
```

Comment:
there's no vector image inside the `img` folder

<img width="350" alt="Screen Shot 2021-11-11 at 9 31 46 AM" src="https://user-images.githubusercontent.com/47992412/141257085-3df6636d-6450-42ab-9a45-45a7c5d78d0a.png">